data/json/items/ammo.json

```diff
@@ -380,7 +380,7 @@
     "color": "dark_gray",
     "description": "Plastic jacketed copper cable of the type used in small electronics.",
     "material": [ "plastic", "copper" ],
-    "//": "Calculated off of copper wire with an extra 0.5mm of diameter and 0.66g.",
+    "//": "Calculated off of copper wire with an extra 0.5 mm of diameter and 0.66g.",
```

Comment:
0.66 g?

Reply:
Thanks for catching that. I also updated it to be 600 mg for milligrams as opposed to fraction of grams.